jenkins-slaves/nodejs10-angular/Dockerfile.rhel7

```diff
@@ -35,7 +35,7 @@ RUN yum -y install /root/google-chrome-stable_current_x86_64.rpm && \
 # Install cypress dependencies
 # Please note: xorg-x11-server-Xvfb is not available on RHEL via yum anymore, so "RUN yum install -y xorg-x11-server-Xvfb" won't work.
 #   Therefore this Dockerfile uses the version from CentOS instead.
-ADD http://mirror.centos.org/centos/7/os/x86_64/Packages/xorg-x11-server-Xvfb-1.20.1-3.el7.x86_64.rpm /root/xorg-x11-server-Xvfb.x86_64.rpm
+ADD http://mirror.centos.org/centos/7/os/x86_64/Packages/xorg-x11-server-Xvfb-1.20.4-7.el7.x86_64.rpm /root/xorg-x11-server-Xvfb.x86_64.rpm
```

Comment:
change this to http://mirror.centos.org/centos/7.7.1908/os/x86_64/Packages/xorg-x11-server-Xvfb-1.20.4-7.el7.x86_64.rpm otherwise we well have the same issue again, if Xvfb version changes in one of the next centos7 updates

Reply:
should be fixed now! thanks!